app/views/admins/pods.mobile.haml

```diff
@@ -0,0 +1,9 @@
+.container
+  .row
+    .col-md-9
+      %h2
+        = t(".pod_network")
+
+      .alert.alert-warning
+        = t(".pod_desktop_view")
+        = link_to t(".pod_desktop_link"), toggle_mobile_path
```

Comment:
```
= t(".pod_desktop_view", desktop_link: link_to(t(".pod_desktop_link", toggle_mobile_path))
```
or
```
!= t(".pod_desktop_view", desktop_link: link_to(t(".pod_desktop_link", toggle_mobile_path))
```
(I didn't try this) and change the translation to

```
pod_desktop_view: "This page is not available on mobile view, please switch to %{desktop_link}."
```

Please don't concatenate translated strings like this. It gets messy in some languages.